nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/test/CoreProviderTest.java

```diff
@@ -171,25 +171,65 @@ public void testSimplePageProviderWithParams() throws Exception {
 
     }
 
-    @Test
     public void testDirectNXQL() throws Exception {
```

Comment:
Is this test deactivated on purpose? This seems wrong.

Reply:
🤦

Reply:
Good catch.  Thank you 👍